hack/cnbi-package-list.yaml

```diff
@@ -0,0 +1,159 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: cnbi-package-list
```

Comment:
could you integrate this pipeline into the reconciliation loop, so that it gets actually run?

Reply:
that's planned in #107